From 2a6ccc0f3d63822c4c84db3bcab78a755387e7fd Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Tue, 30 Jan 2024 09:33:39 +0100
Subject: [PATCH] Rollback definition of :nth() to please parser

Rolling back pending syntax update for `<an+b>` in CSS Syntax spec:
https://github.com/w3c/csswg-drafts/pull/9480

Rollback is partial, keeping the `:nth` prefixing.
---
 ed/css/css-gcpm.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ed/css/css-gcpm.json b/ed/css/css-gcpm.json
index 10b1c649d3..a85186c578 100644
--- a/ed/css/css-gcpm.json
+++ b/ed/css/css-gcpm.json
@@ -133,7 +133,7 @@
   "selectors": [
     {
       "name": ":nth()",
-      "value": ":nth( <an+b> [of <custom-ident>]? )",
+      "value": ":nth( An+B [of <custom-ident>]? )",
       "href": "https://drafts.csswg.org/css-gcpm-3/#selectordef-nth"
     }
   ],
-- 
2.42.0.windows.2

